packages/ckeditor5-image/src/imageutils.ts

```diff
@@ -166,6 +212,13 @@ export default class ImageUtils extends Plugin {
 		return this.isImage( selectedElement ) ? selectedElement : selection.getFirstPosition()!.findAncestor( 'imageBlock' );
 	}
 
+	/**
+	 * Returns an image widget editing view based on the passed image view.
+	 */
+	public getImageWidgetFromImageView( imageView: ViewElement ): ViewContainerElement | null {
+		return imageView.findAncestor( { classes: IMAGE_WIDGETS_CLASSES_MATCH_REGEXP } ) as ( ViewContainerElement | null );
```

Comment:
It will match class like `abcimagexyz`. See my comment for `IMAGE_WIDGETS_CLASSES_MATCH_REGEXP`.

Reply:
True. I changed it.